src/NuGet.Core/NuGet.Protocol/HttpSource/HttpSourceAuthenticationHandler.cs

```diff
@@ -25,7 +25,9 @@ public class HttpSourceAuthenticationHandler : DelegatingHandler
         private readonly HttpClientHandler _clientHandler;
         private readonly ICredentialService _credentialService;
 
+#pragma warning disable CA2213 // Disposable fields should be disposed
```

Comment:
Was this included by mistake?  It conflicts with https://github.com/NuGet/NuGet.Client/pull/4812/files#diff-3bf20d0e4e245fa4c8f535e7ac6c7a7a9567d27c0318919b29308f15c817b028

Reply:
yes, yes it was.